prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -862,5 +862,24 @@ public async Task<ActionResult<EnrolleeNoteViewModel>> DeleteEnrolleeNotificatio
 
             return Ok();
         }
+
+        // GET: api/Enrollees/emails
+        /// <summary>
+        /// Gets all of the enrollee emails that match the type
+        /// </summary>
+        /// <param name="bulkEmailType"></param>
+        [HttpGet("emails", Name = nameof(GetEnrolleeEmails))]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
```

Comment:
This endpoint doesn't return status code 400, but it does implicitly return 403.